test/resources/tzbtc-parameter-entrypoints-ref.mtz

```diff
@@ -1,34 +1,35 @@
-                                                                                          nat
-                                                                                          (map
-                                                                                               string
-                                                                                               string))))))))
+parameter (or (or (or (pair %getVersion (unit %viewParam)
+                                        (contract %viewCallbackTo nat))
+                      (pair %getAllowance (pair %viewParam (address :owner)
+                                                           (address :spender))
+                                          (contract %viewCallbackTo nat)))
+                  (or (pair %getBalance (address :owner %viewParam)
+                                        (contract %viewCallbackTo nat))
+                      (or (pair %getTotalSupply (unit %viewParam)
+                                                (contract %viewCallbackTo nat))
+                          (pair %getTotalMinted (unit %viewParam)
+                                                (contract %viewCallbackTo nat)))))
+              (or (or (pair %getTotalBurned (unit %viewParam)
+                                            (contract %viewCallbackTo nat))
+                      (pair %getOwner (unit %viewParam)
+                                      (contract %viewCallbackTo address)))
+                  (or (pair %getRedeemAddress (unit %viewParam)
+                                              (contract %viewCallbackTo address))
+                      (or (pair %getTokenMetadata (list %viewParam nat)
+                                                  (contract %viewCallbackTo (list (pair
+                                                                                        (nat %metadata_tokenId)
+                                                                                        (pair
+                                                                                              (string %metadata_symbol)
+                                                                                              (pair
+                                                                                                    (string %metadata_name)
+                                                                                                    (pair
+                                                                                                          (nat %metadata_decimals)
+                                                                                                          (map %metadata_extras
+                                                                                                                                string
+                                                                                                                                string))))))))
```

Comment:
The only change here is the addition of some field annotations, right? That sounds fine.

We can use [this feature](https://gitlab.com/morley-framework/morley/-/merge_requests/589) to make metadata annotations consistent with [FA2](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#token-metadata).

Reply:
Yes, that is the only change.

Interesting that we want something to comply with something concrete but do not have tests for exactly that like e.g. `approvableLedgerSpec`.

Reply:
Oh wait, why am I using my own `TokenMetadata` in the first place :thinking: 

I guess I will replace it with ManagedLedger's one.

Reply:
There is no TokenMetadata in ManagedLedger IIRC. It's a concept from FA2. But you can try using it from FA2Interface.

Reply:
Yep, right